process_testsuite.go

```diff
@@ -44,6 +44,14 @@ func (v *Venom) runTestSuite(ctx context.Context, ts *TestSuite) {
 		ts.Vars.Add(k, computedV)
 	}
 
+	exePath, err := os.Executable()
+	if err != nil {
+		log.Errorf("failed to get executable path: %v", err)
+	} else {
+		ts.Vars.Add("venom.executable", exePath)
+	}
+
+	ts.Vars.Add("venom.libdir", v.LibDir)
```

Comment:
Can you add venom.libdir and venom.executable to "Builtin variables" on README please?

Reply:
done :+1: